sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -8,32 +8,51 @@ toc::[]
 
 [role="_abstract"]
 
-{sandboxed-containers-first} support for {product-title} provides users with built-in support for running Kata Containers as an additional optional runtime. This is particularly useful for users who are wanting to perform the following tasks:
+{sandboxed-containers-first} support for {product-title} provides you with built-in support for running Kata Containers as an additional optional runtime. The new runtime supports containers in dedicated virtual machines (VMs), providing improved workload isolation. This is particularly useful for performing the following tasks:
 
+Run privileged or untrusted workloads:: {sandboxed-containers-first} (OSC) makes it possible to safely run workloads that require specific privileges, without having to risk compromising cluster nodes by running privileged containers. Workloads that require special privileges include:
+* Workloads that require special capabilities from the kernel beyond the default ones granted by standard container runtimes such as CRI-O, for example to access low-level networking features.
+* Workloads that need elevated (root) privileges, for example to access a specific physical device. With {sandboxed-containers-first}, the specific device they use (and only that device) can be passed through to the VM, ensuring that the workload cannot access or misconfigure the rest of the system.
```

Comment:
IBM style suggests to avoid using parentheses when possible. I think that the first sentence still conveys the correct meaning when it reads: 
"Workloads that need elevated root privileges, for example to access a specific physical device.

Reply:
Accepted. Removing parenthesis where possible.